
From: Dipankar Sarma <dipankar@in.ibm.com>

OK, yet another mistake. Add fastcall directive to call_rcu().

Signed-off-by: Dipankar Sarma <dipankar@in.ibm.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/rcupdate.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN kernel/rcupdate.c~rcu-no-arg-fastcall-fix kernel/rcupdate.c
--- 25/kernel/rcupdate.c~rcu-no-arg-fastcall-fix	2004-06-22 11:27:41.878391280 -0700
+++ 25-akpm/kernel/rcupdate.c	2004-06-22 11:27:41.882390672 -0700
@@ -74,7 +74,8 @@ static DEFINE_PER_CPU(struct tasklet_str
  * The read-side of critical section that use call_rcu() for updation must 
  * be protected by rcu_read_lock()/rcu_read_unlock().
  */
-void call_rcu(struct rcu_head *head, void (*func)(struct rcu_head *rcu))
+void fastcall call_rcu(struct rcu_head *head,
+				void (*func)(struct rcu_head *rcu))
 {
 	int cpu;
 	unsigned long flags;
_
